src/App.js

```diff
@@ -22,38 +22,33 @@ const App = () => {
   const [hand, setHand] = useState([])
 
   const drawCard = () => {
-    // 1) we are creating a function called drawCard so we can put conditions
+    // 1)
     const draw = `${rank[Math.floor(Math.random() * rank.length)]} ${
       suit[Math.floor(Math.random() * suit.length)]
     }`
-    // 2)  the code below explains that everytime we draw a card, it is randomized so we don't get the suit and rank in that same order. So the rank and suit are plugged in the system and it randomizes by the .random and the .length plugs in the whole array. in this coding, we are getting 1 card from draw that is why there is a -1. so it is subtracting 1 suit and rank from the array.
+    // 2)
     if (hand.indexOf(draw) === -1) {
-      // 3)in this coding, we are getting 1 card from draw that is why there is a -1. so it is subtracting 1 suit and rank from the array. this coding explains that we are creating a new array from "hand" and adding new values at the end of the array
+      // 3)
       setHand([...hand, draw])
-      // 4) this coding explains that we are creating a new array from "hand" and adding new values at the end of the array
+      // 4)
     } else if (hand.length !== 52) {
-      // 5)the array of hand is checking if it not equal to 52. I believe it will stop drawing when it reaches the end of the draw. draws a card and update the hand
+      // 5)
       drawCard()
-      // 6)draws a card and update the hand 
+      // 6)
     } else {
-      // 7)this else statement explains that when the 52 cards have been drawn, it would alert us that "All cards have been dealt"
+      // 7)
       alert("All cards have been dealt.")
     }
   }
   // No need to change anything ABOVE this line ^
 
-  const shuffle = () => {
-    const shuffleArray =[...hand]
-    setHand([])
-  }
```

Comment:
I have seen this same approach a couple of times from the cohort. I get the thinking here but shuffleArray isn't actually doing anything. The randomization of the cards happens on the draw so really the only thing needed is to reset the initial state.